doc/rustaceanvim.txt

```diff
@@ -250,10 +250,22 @@ rustaceanvim.code-action.Opts                    *rustaceanvim.code-action.Opts*
 
     Fields: ~
         {group_icon?}          (string)
-                                          Text appended to a group action
+                                                                Text appended to a group action
```

Comment:
👀

Reply:
This file is generated using [vimcats](https://github.com/mrcjkb/vimcats).

I still have to implement [a line length limitation](https://github.com/mrcjkb/vimcats/issues/15), but haven't had time yet.